_1327/minutes/tests.py

```diff
@@ -71,7 +71,7 @@ def test_publish_button(self):
 		document = mommy.make(MinutesDocument, participants=self.participants, moderator=self.moderator, state=MinutesDocument.UNPUBLISHED)
 		document.set_all_permissions(staff_group)
 
-		self.app.get(reverse('documents:publish', args=[document.url_title]), user=self.user)
+		self.app.get(reverse('documents:publish', args=[document.url_title, 1]), user=self.user)
```

Comment:
This is the state id not a group id. The states are not easily accessible in the template so hard coding is okay.